### PR TITLE
Source Zendesk Chat: Add conversion and department events streams

### PR DESCRIPTION
**Pull request by am6010**
## What
2 extra streams for the zendesk-chat connector. 
- conversions 
- department events
Both of them are time incremental. 

[Issue Ticket](https://github.com/airbytehq/airbyte/issues/19641)

## How
Use the `TimeIncrementalStream` streams to read events from `conversion` and `deparment_events` endpoints.

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.
No breaking change.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*
Screenshot 2022-12-13 at 6 48 26 PM

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*
Screenshot 2022-12-13 at 6 40 23 PM

</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Hello :wave:, first thank you for this amazing contribution. 

We really appreciate the effort you've made to improve the project.
We ask you patience for the code review. Last month our team was focused on Hacktoberfest event and that probably left some PR without the proper feedback. And this week, due to the Thanksgiving US Holiday, most our team is out of office with their families. Another important piece of information why code won't be merge this week is: as a safety measure the core team has decided to freeze merging code to main branch to keep the release stable. Next week we'll return to you with the proper code review and update the status of your contribution.

If you have any questions feel free to send me a message in Slack!
Thanks!

**Comment by smrutiparida** [comment]
@am6010 Could you please update the information required in the pre-merge checklist

- issue # this PR is fixing
-  Unit test output and screenshot
- Acceptance test output and screenshot
- Integration test output and screenshot
- Add tests in unit_tests/test_streams
- Add configuration entries in the configured_catalog.json file for the integration_tests
- Bump the version in Dockerfile 

It shall also be helpful to add the QA steps for this PR.

File: airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/schemas/conversions.json
Comment (bazarnov) [inline airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/schemas/conversions.json:3]: Please add `"additionalProperties": true` to the root of all schemas newly added to support backward capability with all previous versions

File: airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/schemas/department_events.json
Comment (bazarnov) [inline airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/schemas/department_events.json:4]: Same comment about `"additionalProperties": true`

**Review by bazarnov — APPROVED** [review]
@am6010 @marcosmarxm 

LGTM, once `source-acceptance-test` is passed and comments are considered. Thanks for the contribution)

**Comment by am6010** [comment]
@bazarnov Thank you for reviewing. I have added the suggested changes. 
Can you please trigger the acceptance tests?

**Comment by bazarnov** [comment]
> @bazarnov Thank you for reviewing. I have added the suggested changes. Can you please trigger the acceptance tests?

Please add these: 'conversions' and 'department_events' streams to the list of `empty_streams` inside the `source-acceptance-test` config, as far as we don't have that data populated for our sandbox and SAT fails on this.

**Comment by am6010** [comment]
Hi,  @smrutiparida I have added the screenshots and done the change @bazarnov  asked for.

**Comment by smrutiparida** [comment]
LGTM! Thanks

**Comment by bazarnov** [comment]
@marcosmarxm are you available to assist here? I think we are good to go with merge here, the final check on your side is needed.

**Comment by marcosmarxm** [comment]
Hello :wave::skin-tone-2: and thank you for your contribution!

Airbyte has instituted a code freeze between 19 and 30 December, to make sure there are no disruptions during the holidays. 
Because of this, reviewing and merging your contribution may take longer than usual. 
We apologize for the delay, but we want everyone to have a quiet and happy holiday.

If you have any questions or need further clarification, please don't hesitate to ping via Slack.

File: airbyte-webapp/src/components/JobItem/components/JobLogs.tsx
Comment (bazarnov) [inline airbyte-webapp/src/components/JobItem/components/JobLogs.tsx:56]: Please remove this file from your PR.

File: airbyte-webapp/src/components/JobItem/components/MainInfo.tsx
Comment (bazarnov) [inline airbyte-webapp/src/components/JobItem/components/MainInfo.tsx:20]: Please remove this file from your PR.

File: airbyte-webapp/src/views/Connector/ConnectorForm/components/Sections/auth/AuthButton.test.tsx
Comment (bazarnov) [inline airbyte-webapp/src/views/Connector/ConnectorForm/components/Sections/auth/AuthButton.test.tsx:74]: Please remove this file from your PR.

File: airbyte-webapp/src/views/Connector/ConnectorForm/index.stories.tsx
Comment (bazarnov) [inline airbyte-webapp/src/views/Connector/ConnectorForm/index.stories.tsx:6]: Please remove this file from your PR.

**Review by bazarnov — CHANGES_REQUESTED** [review]

File: airbyte-webapp/src/components/JobItem/components/JobLogs.tsx
Comment (am6010) [reply airbyte-webapp/src/components/JobItem/components/JobLogs.tsx]: I think the repo has some github actions that added these changes.  I have reverted them.

**Comment by am6010** [comment]
hey @bazarnov and @marcosmarxm can you please trigger the acceptance tests again?

**Comment by sh4sh** [comment]
Hoping to merge https://github.com/airbytehq/airbyte/pull/23876 first to fix the changelog and then I'll fix the merge conflicts.

**Comment by sh4sh** [comment]
hey @am6010 I resolved the merge conflicts but I do not have push access on your fork. I have added these changes in https://github.com/airbytehq/airbyte/pull/23878/ can you sync them to this PR?

Running something like this from your working branch should do:

1. Pull changes from my airbytehq/airbyte branch:
`git pull git@github.com:airbytehq/airbyte.git source-zendesk-chat/conversion_and_department_evets_streams`

2. Push changes to your fork (assuming it is configured as default `origin`)
`git push origin source-zendesk-chat/conversion_and_department_evets_streams`

**Comment by am6010** [comment]
@sh4sh Thank you for your help. I have pulled your changes into the branch. Let me know if there is an issue with the tests.

**Pull request closed without merging**
